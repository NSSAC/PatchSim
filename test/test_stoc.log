2018-08-09 11:54:03,320 - patchsim - INFO - Starting PatchSim
2018-08-09 11:54:03,322 - patchsim - INFO - Loaded patch attributes
2018-08-09 11:54:03,322 - patchsim - INFO - Loaded disease parameters from Config
2018-08-09 11:54:03,322 - patchsim - INFO - No ParamFile loaded
2018-08-09 11:54:03,329 - patchsim - INFO - Loaded temporal travel matrix
2018-08-09 11:54:03,333 - patchsim - INFO - Loaded seeding schedule
2018-08-09 11:54:03,403 - patchsim - INFO - Initializing simulation run...
2018-08-09 11:54:03,403 - patchsim - INFO - Found RandomSeed. Running in stochastic mode...
2018-08-09 11:54:03,433 - patchsim - INFO - Simulation complete. Writing outputs...
2018-08-09 11:54:03,435 - patchsim - INFO - Done! Time elapsed: 0.11401104927062988 seconds.
