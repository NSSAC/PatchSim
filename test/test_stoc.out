A 20 10 7 5 8 10 11 9 8 17 16 22 22 26 21 35 25 42 59 65 59 77 86 84 89 114 116 128 146 142 143 183 174 186 201 213 195 207 212 233 228 240 233 227 221 203 227 188 168 165 157 138 117 116 90 88 63 64 57 53 56 38 34 21 28 28 22 20 14 18 13 4 8 2 6 5 5 3 2 1 1 3 0 0 1 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
B 20 8 13 4 12 9 10 9 20 21 15 14 26 28 36 29 31 47 39 60 54 67 65 74 91 100 112 98 121 155 146 133 183 174 155 240 215 195 236 223 236 200 219 191 171 188 174 159 172 150 167 138 134 128 104 115 78 80 60 58 55 44 34 31 26 30 26 19 19 19 13 6 9 8 5 6 7 7 6 5 4 3 2 0 0 6 2 3 0 2 0 2 0 2 0 0 0 0 0 1 0
C 0 20 42 31 35 19 39 42 48 47 41 49 59 82 82 89 87 109 112 130 152 172 179 211 215 227 231 266 238 280 254 312 280 250 257 199 232 174 154 172 141 127 119 80 94 68 77 59 44 33 39 29 31 24 27 16 15 15 15 8 6 10 4 11 1 5 4 2 3 2 0 3 0 0 1 0 0 1 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
